src/PyCall.jl

```diff
@@ -429,7 +439,7 @@ function pyimport(name::AbstractString)
     if ispynull(o)
         if pyerr_occurred()
             e = PyError("PyImport_ImportModule")
-            if e.T.o == @pyglobalobjptr(:PyExc_ImportError)
+            if PyPtr(e.T) == @pyglobalobjptr(:PyExc_ImportError)
```

Comment:
I feel like we should just define an `==` method for `PyObject == PyPtr`.

Reply:
Or maybe call it something else, since this isn't really equality.  Maybe `≛` (`\starequal`)

Reply:
Done. (But only substituted it in the comparison above so far)